hs_composite_resource/models.py

```diff
@@ -32,11 +32,18 @@ def set_default_logical_file(self):
         of the resource that is not already associated with a logical file. """
 
         for res_file in self.files.all():
+            # TODO: This following if block will not be needed once we migrate
+            # raster file type to HStore
```

Comment:
@hyi Note the review feedback that I need is on the design aspect of file type metadata management using key/value storage. This is a WIP PR.